File: scala/darts/src/main/scala/Darts.scala
object Darts {
  def score(x:Double,y:Double): Int = {
    val r=Math.sqrt(Math.pow(x,2) + Math.pow(y,2))
    if(r<=1) 10
    else if(r<=5) 5
    else if(r<=10) 1
    else 0

  }
}

File: julia/darts/darts.jl
# the dart exercise

function score(x, y)
    # you can do lambda like things, which is nice
    norm(x1, y1) = sqrt(sum(x1^2 + y1^2))

    # you can do short circuit style operations
    norm(x, y) <= 1 && return 10
    norm(x, y) <= 5 && return 5
    norm(x, y) <= 10 && return 1

    # and if none of the short circuits matched, return 0
    return 0
end
